internal/util/function/text_embedding_function.go

```diff
@@ -98,25 +99,26 @@ func NewTextEmbeddingFunction(coll *schemapb.CollectionSchema, functionSchema *s
 	var embP textEmbeddingProvider
 	var newProviderErr error
 	conf := paramtable.Get().FunctionCfg.GetTextEmbeddingProviderConfig(base.provider)
+	credentials := credentials.NewCredentialsManager(paramtable.Get().CredentialCfg.GetCredentials())
```

Comment:
Let's make it a global singleton

Reply:
Considering the needs on the cloud, this object is not suitable as a global singleton

Reply:
To detect changes in milvus.yaml, the singleton can only be an empty object